rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/CommandLineBuilder.java

```diff
@@ -252,8 +264,14 @@ public String getSourceName() {
      * @throws java.io.IOException if you supplied a ProActive Configuration file that doesn't exist.
      */
     public String buildCommandLine(boolean displayCredentials) throws IOException {
-        List<String> command = this.buildCommandLineAsList(displayCredentials);
-        return Joiner.on(' ').join(command);
+        if (!deploymentMode.equals("NoStartup")) {
```

Comment:
To confirm, `NoStartup` means no extra startup script from the user? Could you please comment it?